### PR TITLE
Add the community page for the website

### LINKED ISSUE #13: Add a community page
I am  suggesting to make a community page which lists all ways people can interact with the community or get news. At the moment we only have social media buttons at the bottom of page. Having a separate page would allow being more verbose and list everything including the channels for subprojects and also including information like what is bridged with what. It would also allow adding a short description for what's the preferred place for different kind of discussions.

Examples
* https://godotengine.org/community
* https://www.haiku-os.org/community/
* https://inkscape.org/community/
* https://www.rust-lang.org/community

### PR DESCRIPTION
**Pull request by hstr2785**
This Pull Request adds the Community page for the website.
I have added the particulars in the `config.yml` to create the URL, created a file named `community.md`  and community.html to make this happen.
I have followed the layout in which the page organisation is written and the page looks like this: 
image

image

As we have two IRC channels, I wasn't sure about what to link to on the button underneath the information written about them. We can have two buttons if we want to.

Initially, I had seen the SVGs for different icons on the home page and had used that while working on this, but later I saw how the  `organization`  page was written and switched this to make it look like that. Here's how that looked:

image

I have attached this screenshot just to show that we can do this, if we want to.

I hope I have followed the coding styles.
Feel free to request changes anything.
Fixes #13

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ menu:
       - name: Organization
         url: /organization/
         weight: 10
+      - name: Community
+        url: /community/
+        weight: 15
```
Comment (ITAYC0HEN) [inline config.yml:25]: I wonder if it better be in the toolbar or a sub-page in the Organization page.
So either `Home -> Community` or `Home -> Orgnization -> Community`.

What do you think is better?

Comment (hstr2785) [reply config.yml:25]: Thank you for your swift reply. Considering the fact that all the sample links on the issue have a separate community page that is listed in the toolbar, I think will be a great idea to list it as a separate page. The organization page also has a community. So, that's why I came into such a conclusion. I'm definitely open to suggestions from others.

File: layouts/_default/community.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  </p>
+</div>
+<div class="grid_parent">
+  <h1 class="mb-15 page-subtitle">Connect</h1>
+  <div class="teams-section">
+    <div class="team-item">
+      <div>
+        <h3><mark>Mattermost</mark></h3>
+        <p>
+          Here, we have separate channels for different teams, sub-projects and development topics.
```
Comment (ITAYC0HEN) [inline layouts/_default/community.html:22]:
```suggestion
          <b>Recommended!</b> This is our main hub which is separated into channels for different teams, sub-projects, and development topics.
```

File: layouts/_default/community.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      </div>
+    </div>
+    <div class="team-item">
+      <div>
+        <h3><mark>Telegram</mark></h3>
+        <p>
+          This is your place to have discussions about Rizin, ask questions and get support.
+        </p>
+        <p>
+          This is bridged to the General channel of Mattermost.
```
Comment (ITAYC0HEN) [inline layouts/_default/community.html:36]:
```suggestion
          The Telegram group is a bridge from Mattermost's #General channel to Telegram. And it is intended for user support and general discussions related to Rizin.
```

File: layouts/_default/community.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      </div>
+    </div>
+    <div class="team-item">
+      <div>
+        <h3><mark>IRC</mark></h3>
+        <p>
+          On Freenode (IRC), you can join #rizin for general discussions and #rizindev for questions related to development. 
+        </p>
+        <p>
+          Both are bridged to the General channel and Rizin Dev channel on Mattermost.
```
Comment (ITAYC0HEN) [inline layouts/_default/community.html:50]:
```suggestion
          Both are respectively bridged to the General and Rizin Dev channels on Mattermost.
```

File: layouts/_default/community.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+      </div>
+      <div class="btn-team-members">
+        <a class="site-button btn-secondary" href="https://t.me/rizinorg">Join us on Telegram</a>
+      </div>
+    </div>
+    <div class="team-item">
+      <div>
+        <h3><mark>IRC</mark></h3>
+        <p>
+          On Freenode (IRC), you can join #rizin for general discussions and #rizindev for questions related to development.
```
Comment (ITAYC0HEN) [inline layouts/_default/community.html:47]:
```suggestion
          On Freenode (IRC), you can join #rizin for general discussions and #rizindev for questions and discussions related to Rizin development. 
```

File: layouts/_default/community.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  </div>
+</div>
+<div class="grid_parent">
+  <h1 class="mb-15 page-subtitle">Social</h1>
+  <div class="teams-section">
+    <div class="team-item">
+      <div>
+        <h3><mark>Twitter</mark></h3>
+        <p> 
+          Join us on Twitter for release announcements and much more.
```
Comment (ITAYC0HEN) [inline layouts/_default/community.html:66]:
```suggestion
          Follow us on Twitter for release announcements and much more.
```

**Review by ITAYC0HEN — CHANGES_REQUESTED** [review]
Hey @hstr2785! Thank you for this great page, it looks terrific, really love it!
I left some comments and ideas for improvements of the text.

Regarding the icons, I think that the version without the icons looks better, although I would give it a try to see how it looks like when the icons are inside the action buttons ("[icon]    Join us on..."), maybe it will look better.


Really great work, thank you!

**Comment by hstr2785** [comment]
Adding the SVG in the button doesn't really look great. Since the size of SVG becomes really small, I don't think it makes a difference. This is the best I could do :

image

Also regarding the IRC channels since you have a couple of them, should I add separate buttons for the respective channels or will the single button there leading to one of the IRC be enough. Here is the double button in action:

image

Otherwise, we can have it on two regular sized buttons, like from the Rust Lang's page: [link](https://www.rust-lang.org/community)

**Comment by ITAYC0HEN** [comment]
Regardin the icons, you are right, it does to look good and we better not to use icons at all for now.
The question about IRC is a great question! I think that "#rizindev" can be a link and then we will have only a single button "Join us on IRC". In turn, in IRC chat description itself (not the website) we will make sure to put a link to the dev channel (if it is not already therE).

**Comment by hstr2785** [comment]
As you had specified earlier I have added the links of the IRC channels in the description. The button in the screenshot leads to the "#rizin" channel in IRC. Here is the preview :
 
image

**Review by ITAYC0HEN — APPROVED** [review]
Thank you! I think it looks good :)
If there is nothing else to add, please mark the PR as Ready. I think it can be merged then 🎉

**Comment by ITAYC0HEN** [comment]
Congratulations! It is up and running -> https://rizin.re/community/

Would you like to send another PR and make all the links in this page to be opened in a new tab? (`target="_blank"`)

**Comment by hstr2785** [comment]
Thank you for accepting this merge request. 
As specified, I have made changes to the links. Please review #19 next time you are free.
Please don't hesitate to mention me if there is anything that needs to be done.

**Pull request merged**